store-collected-images/webextension-plain/deps/idb-file-storage.js

```diff
@@ -4,7 +4,7 @@
   } else if (typeof exports !== "undefined") {
     factory(exports);
   } else {
-    var mod = {
+    let mod = {
```

Comment:
I think anything that is third party dependency for a webextensions-example (as in "coming from an outside repo") should be kept out of this change and left unmodified (and as a side note this part is not even part of that library, it is actually code injected automatically as part of the bundling).